src/components/SingleTermView/OverView/Details.jsx

```diff
@@ -0,0 +1,157 @@
+import {
+  Box,
+  Chip,
+  Grid,
+  Stack,
+  Typography
+} from "@mui/material";
+import { vars } from "../../../theme/variables";
+import React from "react";
+
+const { gray800, gray500 } = vars;
+const Details = () => {
+  const synonyms = [
+    { title: 'CNS', description: 'abbrev' },
+    { title: 'Myencephalon', description: '' },
+    { title: 'Cerebrospinal axis', description: 'fma:synonym' },
+    { title: 'Cerebrospinal axis', description: 'fma:synonym' },
+    { title: 'Central nervous system', description: 'fma:synonym' },
+    { title: 'systema nervosum centrale', description: 'oboInOwl:hasExactSynonym' },
+    { title: 'neuraxis', description: 'oboInOwl:hasRelatedSynonym' },
+  ];
+  
+  const existingIDs = [
+    'UBERON:0001017',
+    'FMA:55675',
+    'ILX:0101901',
+    'BIRNLEX:1099',
+  ];
+  
+  return (
+    <>
+      <Grid container>
+        <Grid item xs={12} lg={6}>
+          <Stack spacing=".75rem">
+            <Typography color={gray800} fontWeight={500}>
+              Synonyms
+            </Typography>
+            <Box display="flex" flexWrap="wrap" gap=".5rem">
+              {synonyms.map((synonym) => (
+                <Chip
+                  className="rounded synonyms"
+                  variant="outlined"
+                  key={synonym.title}
+                  label={
+                    <span>
+                        {synonym.title} <span>{synonym.description}</span>
+                      </span>
+                  }
+                />
+              ))}
+            </Box>
+          </Stack>
+        </Grid>
+        <Grid item xs={12} lg={2}>
+          <Stack spacing=".75rem">
+            <Typography color={gray800} fontWeight={500}>
+              Preferred ID
+            </Typography>
+            <Typography fontSize=".875rem" color={gray500}>
+              UBERON:0000955
+            </Typography>
+          </Stack>
+        </Grid>
+        <Grid item xs={12} lg={4}>
+          <Stack spacing=".75rem">
+            <Typography color={gray800} fontWeight={500}>
+              Existing IDs
+            </Typography>
+            <Box display="flex" flexWrap="wrap" gap=".5rem">
+              {existingIDs.map((id) => (
```

Comment:
same principle here, either we move these to the props received or to the get call. thanks